components/jira/ticket-selector.tsx

```
@@ -0,0 +1,289 @@
+              <div className="flex flex-wrap gap-1 max-h-20 overflow-y-auto">
+                {selectedTickets.map((ticket) => (
+                  <Badge
+                    key={ticket.id}
+                    variant="secondary"
+                    className="flex items-center gap-1"
+                  >
+                    <span className="font-mono text-xs">{ticket.issueKey}</span>
+                    <button
+                      onClick={() => onTicketDeselect?.(ticket)}
+                      className="ml-1 hover:bg-gray-200 rounded-full p-0.5"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)